src/amdgpu_libdrm.h

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#define LIBDRM_SAMPLE_DELAY 3500
+#define LIBDRM_SAMPLE_BUF_SIZE 512
+
+#define LIBDRM_GRBM_STATUS 0x8010
+
+enum LIBDRM_GRBM_BITS {
+    LIBDRM_GRBM_BUSY_BIT = 1U << 31
+};
+
+struct libdrm_sample {
+    bool busy_bit;
+};
+
+struct libdrm_stats {
+    int busy;
+};
+
+extern std::string dri_device_path;
+extern bool do_libdrm_sampling;
+
+void libdrm_do_sample();
+void libdrm_thread();
+void libdrm_get_info();
+int libdrm_initialize();
```

Comment:
Of the 4 above, only `libdrm_get_info()` should remain here. The rest ought to be annotated as static within the .C file.

Might need to move `libdrm_get_info` within the bottom of the (C) file to avoid forward declarations.